template/page-dev-tools.html.jinja

```diff
@@ -0,0 +1,193 @@
+{% extends "base.html.jinja" %}
+
+{% block bodyclasses %}no-sidebar{% endblock %}
+{% block mainclasses %}page-dev-tools{% endblock %}
+
+{% block main %}
+
+<div class="">
+  <section class="py-20">
+    <div class="mx-auto text-lg-left">
+      <div class="d-flex flex-column-reverse">
+        <p class="mb-0">
+          {% trans %}Use the developer tools to test, explore, and validate XRP Ledger
+          API requests and behavior. {% endtrans %}
+        </p>
+        <h3 class="eyebrow mb-3"> {% trans %} Dev Tools  {% endtrans %}</h3>
+      </div>
+    </div>
+  </section>
+
+
+  
+  <section class="py-10">
+    <div class="mx-auto text-lg-left">
+      <ul class="nav nav-tabs pb-15" id="myTab" role="tablist">
```

Comment:
Don't use placeholder IDs like `myTab`. I think you don't need an ID here at all, actually.

```suggestion
      <ul class="nav nav-tabs pb-15" role="tablist">
```